api/routes/notes/get-note.routes.js

```diff
@@ -167,4 +167,32 @@ router.delete(
 	},
 );
 
+// Retrieve note's revisions
+router.get('/users/:username/notes/:noteId', isAuthor, async (req, res) => {
+	const { data, error } = await supabase
+		.from('revisions')
+		.select('createdat, revision_id')
+		.eq('note', req.params.noteId);
+
+	if (error) throw new Error(error.message);
```

Comment:
Can you directly throw the error if one occurs please chief? 🚀😄👨‍💻

Reply:
No problem chief